appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
```

Comment:
So I get that this means that the contents of the `bld\bin` directory are uploaded to the cloud after the build (and presumably the build status page will give a link to that location).

But why do we need to do that? @nguerrera ?